afl-fuzz.c

```diff
@@ -3186,6 +3205,17 @@ static u8 save_if_interesting(char** argv, void* mem, u32 len, u8 fault) {
   s32 fd;
   u8  keeping = 0, res;
 
+  u32 cksum = hash32(trace_bits, MAP_SIZE, HASH_CONST);
```

Comment:
@jiradeto Thanks! What I had in mind was simpler than the "fuzz" from AFLFast. For each **seed** S  (i.e., selected input) we would track how often it is fuzzed/mutated. In contrast, "fuzz" tracks for each seed S' how often you explore the same path as S' when fuzzing seed S. Does that make sense?

Reply:
@wuestholz Oh I see! We are not using the same criteria as in AFLFast. Thank you for your explanation. It was fixed now.

Reply:
@jiradeto Great! Thanks!

Reply:
@jiradeto Could you please also remove the call to `hash32`?